Lib/test/test_enum.py

```diff
@@ -1495,6 +1495,17 @@ class SpamEnum(Enum):
             spam = nonmember(SpamEnumIsInner)
         self.assertTrue(SpamEnum.spam is SpamEnumIsInner)
 
+    def test_using_members_as_nonmember(self):
+        class Example(Flag):
+            A = 1
```

Comment:
Just to really drive it home, either in this case or another, use `auto()`

I think that's where the code got sufficiently separated from what's obvious.

Like this code makes it super obvious, but `auto()` has enough indirection that I forgot these wr just ints (especially considering I didn't use `IntFlag`)

Reply:
ok, why not :) I will add one more test example.